gm4_blossoming_pots/data/gm4_blossoming_pots/guidebook/blossoming_pots.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "blossoming_pots",
+    "name": "Blossoming Pots",
+    "module_type": "module",
+    "icon": {
+      "id": "minecraft:decorated_pot"
+    },
+    "criteria": {
+      "obtain_pot": {
+        "trigger": "minecraft:inventory_changed",
+        "conditions": {
+          "items": [
+            {
+              "items": [
+                "minecraft:decorated_pot",
+                "minecraft:flower_pot"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "sections": [
```

Comment:
You may want to consider adding the list of pottable plants as another section (or the same one). The goal is for them to learn everything using the guidebook instead of the wiki page (some guidebook entries don't follow this tho, so it's fine either way).